### PR TITLE
encfs: test should work in sandbox

### PR DESCRIPTION
**Pull request by ilovezfs**
- punt on the test until a sandboxed version works
- :head needs cmake now
- specify :cxx11 for only < :mavericks

Issue Homebrew/homebrew#50602
Upstream issue vgough/encfs#151

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/encfs.rb [hunk 1]
```diff
@@ -19,24 +22,27 @@ class Encfs < Formula
...
 
   def install
-    ENV.cxx11
-    system "make", "-f", "Makefile.dist"
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}",
-                          "--with-boost=#{HOMEBREW_PREFIX}"
+    ENV.cxx11 if MacOS.version < :mavericks
+    if build.head?
+      mkdir "build"
```
Comment (DomT4) [inline Formula/encfs.rb]: You can do `mkdir "build" do` and kill the next `cd`

Comment (ilovezfs) [inline Formula/encfs.rb]: @DomT4 I think that will leave us in the wrong directory for `make` and `make install`

Comment (DomT4) [inline Formula/encfs.rb]: Oh yeah, it will here, I missed those steps because they weren't part of the highlighted diff.

**Pull request closed without merging**